src/test/java/org/folio/pact/ModConfigurationPactTest.java

```diff
@@ -97,6 +98,7 @@ public RequestResponsePact createPact(PactDslWithProvider builder) {
   }
 
   @Test
+  @Ignore
```

Comment:
Please add //TODO or @Ignore("....") why this test was ignored

Reply:
Done